docs/user-guide/src/troubleshooting.md

```diff
@@ -0,0 +1,72 @@
+# Troubleshooting
+
+## Verify that Ironic and Baremetal Operator are healthy
```

Comment:
Honestly, I liked the FAQ format where each title is how a user would google it. Like, what would prompt an operator to look at this section you're adding?

Reply:
Sure, I also like that format, but I know also that an unhealthy Ironic or BMO could show itself in probably dozens of ways. It is more of a sanity check and something I would expect to direct users to when they are having issues that are hard to pinpoint. So I think it is important to have this somewhere. If you want, we can split this into troubleshooting and FAQ maybe?